src/components/10-atoms/fieldset/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@axa-ch/fieldset",
+  "version": "0.0.0-beta.0",
+  "description": "> TODO: description",
+  "author": "Pattern Warriors",
+  "homepage": "https://github.com/axa-ch/patterns-library/tree/master/src/components/10-atoms/fieldset#readme",
+  "license": "Copyright 2019 AXA Versicherungen AG",
+  "main": "lib/index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "files": [
+    "lib",
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/axa-ch/patterns-library.git"
+  },
+  "bugs": {
+    "url": "https://github.com/axa-ch/patterns-library/issues"
+  },
+  "dependencies": {
+    "@webcomponents/webcomponentsjs": "^2.2.7",
```

Comment:
please remove this, sorry again i just fixed today this issue in scaffolding

Reply:
done